apps/pie-docs/src/accessibility/testing-for-accessibility.md

```diff
@@ -0,0 +1,135 @@
+
+Below are some important factors we test when conducting screen reader testing:
+
+{% list {
+    type: listTypes.icon,
+    iconName: "check-circle-filled",
+    iconFill: "support-positive",
+    items: [
+        "**Compatibility**: The screen reader should be able to navigate through the content, read the text, and interact with the controls.",
+        "**Navigation and Focus**: The screen reader should be able to properly navigate through our product using keyboard commands, landmarks, headings, and other navigation techniques.",
+        "**Reading Order**: The screen reader should read content in a logical and meaningful order. It should follow the intended reading order, especially for complex layouts with multiple columns, tables, or responsive designs.",
+        "**Alternative Text (Alt Text)**: Images should have appropriate alternative text descriptions (alt text) to provide meaningful information to visually impaired users. We verify that alt text is available and properly announced by the screen reader when images are encountered.",
+        "**Form Accessibility**: We ensure that form elements, labels, and error messages are properly associated and announced.",
+        "**ARIA Roles and Attributes**: We use [Accessible Rich Internet Applications](https://www.w3.org/WAI/standards-guidelines/aria/) (ARIA) roles and attributes to enhance the accessibility of dynamic and interactive elements. We use screen reader testing to verify that ARIA roles and attributes are correctly implemented and announced by the screen reader, providing additional context and functionality.",
+        "**Multimedia Accessibility**: Screen reader testing also involves assessing the accessibility of multimedia elements, such as audio and video. We use it to ensure that appropriate alternative text or captions are provided for multimedia content."
+    ]
+} %}
+
+---
+
+## Testing using Applause
+
+We have partnered with [Applause](https://www.applause.com/), a third-party crowdsourcing company, to help us with accessibility testing through a contract.
+
+Through this partnership, we can arrange **one-off specialised audits**, conducted by trained accessibility testers from Applause, for both new and existing features. By doing this, we can help teams **understand the state of accessibility in their current products** and how they can improve them to increase their level of accessibility.
+
+{% notification {
+  type: "information",
+  message: "If your team would like to schedule and audit then [get in touch](/support/contact-us/) with us."
```

Comment:
```suggestion
  message: "If your team would like to schedule an audit then [get in touch](/support/contact-us/) with us."
```

Reply:
@FayeCarter I fixed this in previous comments, so it should be solved now in designs, thanks!